tests_my/pages/dashboard/login_page.py

```diff
@@ -0,0 +1,25 @@
+from selenpy.element.text_box import TextBox
+from selenpy.element.base_element import BaseElement
+from selenpy.element.combo_box import ComboBox
+from tests_my.testcases.utils import constants
+
+
+class LoginPage():
+
+    def __init__(self):
+        self._txt_username = TextBox("id=username")
+        self._txt_password = TextBox("id=password")
+        self._btn_login = BaseElement("class=btn-login")
+        self._cbb_repository = ComboBox("id=repository")
+
+    def fill_info_login(self, username, password, repository):
+        if repository is None:
```

Comment:
there should have wait_for first element of Login page, before perform logging

Reply:
I updated